src/sentry/mediators/sentry_app_installation_tokens/destroyer.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import
+
+from sentry.utils.cache import memoize
+from sentry.mediators import Mediator, Param
+from sentry.models import (
+    AuditLogEntryEvent, SentryAppInstallationToken
+)
+
+
+class Destroyer(Mediator):
+    sentry_app_installation = Param('sentry.models.SentryAppInstallation')
+    api_token = Param('sentry.models.ApiToken')
+    generate_audit = Param(bool, default=False)
+    user = Param('sentry.models.User')
+    request = Param('rest_framework.request.Request', required=False)
+
+    def call(self):
+        self._destroy_sentry_app_installation_token()
+        self._destroy_api_token()
+        return self.api_token
```

Comment:
@MeredithAnya I guess we want to return just the `api_token` here instead of the `sentry_app_installation_token`. Does it make sense to change the creator to do this as well to be consistent?

Reply:
Hmm, yeah I feel like that would be less confusing tbh

Reply:
@MeredithAnya Yea, no problem. I can do that in my API PR.